app/assets/stylesheets/projects/project_comments.scss

```diff
@@ -0,0 +1,29 @@
+@import 'mixins/variables';
+@import 'mixins/clearfix';
+@import 'projects/project_variables';
+
+section#project-comments {
+  background-color: $common-background-color;
+  border-radius: 5px;
+  margin-bottom: 50px;
+  margin-left: $first_column_width;
+  padding-top: 1px;
+  width: $article_column_width;
```

Comment:
> Name of variable `article_column_width` should be written in all lowercase letters with hyphens instead of underscores
> [link](https://sider.review/gh/71619232/pull_requests/37)

これは影響範囲が大きい。やるなら別PRにしたい